lib/src/command/outdated.dart

```diff
@@ -31,32 +31,40 @@ class OutdatedCommand extends PubCommand {
   String get docUrl => 'https://dart.dev/tools/pub/cmd/pub-outdated';
 
   OutdatedCommand() {
-    argParser.addFlag('color',
-        help: 'Whether to color the output. Defaults to color '
-            'when connected to a terminal, and no-color otherwise.');
-
     argParser.addFlag('json',
-        help: 'Outputs the results in a json formatted report',
+        help: 'Outputs the results in a json formatted\n'
+            'report.',
         negatable: false);
 
+    argParser.addFlag('color',
+        help: 'Whether to color the output.\n'
```

Comment:
Maybe "Whether to use color in the output.\n"?

Reply:
Done